### PR TITLE
Note filter

### PR DESCRIPTION
**Pull request by abhisheksahani-nova**

### CODE REVIEW

File: src/pages/Notes/Notes.jsx [hunk 1]
```diff
@@ -80,6 +158,14 @@ function Notes() {
           />
```
Comment (MuskanChhatrasal) [inline src/pages/Notes/Notes.jsx:158]: Why you have used it here? If not in use can remove this.

**Review by MuskanChhatrasal — APPROVED** [review]
Cleaner code with amazing UI. Few suggestions: Properly inclined the code then it will be more readable. Else everything looks good to merge.

File: src/pages/Notes/Notes.jsx (hunk 1 above)
Comment (abhisheksahani-nova) [reply src/pages/Notes/Notes.jsx:158]: I didn't get what your saying, can you specify exactly

Comment (MuskanChhatrasal) [reply src/pages/Notes/Notes.jsx:158]: I think you have used extra closing tag there.

Comment (abhisheksahani-nova) [reply src/pages/Notes/Notes.jsx:158]: Okay , thanks for the review, I will check that

File: src/pages/Notes/FilterModal/FilterModal.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import "./FilterModal.css";
+import { useLabels } from "../../../context/labels-context";
+
+function FilterModal({ filterDispatch, filterState, setOpenFilterModal }) {
+  const { labels } = useLabels();
+
+  return (
+    <div className={`playlist-dropdown-container`}>
```
Comment (harshalini) [inline src/pages/Notes/FilterModal/FilterModal.jsx:9]: The naming could be better, didn't get the use of playlist name

File: src/pages/Notes/Notes.css [hunk 3]
```diff
@@ -2,8 +2,8 @@
   width: 35rem;
   border-radius: 5px;
   padding: 0.8rem;
-  margin-right: 11rem;
-  margin-left: 8rem;
+  margin-right: 10rem;
+  margin-left: 7rem;
```
Comment (harshalini) [inline src/pages/Notes/Notes.css:6]: You can use margin shorthand here

**Review by harshalini — APPROVED** [review]
Code is well structured, well implementation of the filter

File: src/pages/Notes/FilterModal/FilterModal.jsx (hunk 2 above)
Comment (abhisheksahani-nova) [reply src/pages/Notes/FilterModal/FilterModal.jsx:9]: Actually it was prebuild class

File: src/pages/Notes/Notes.css (hunk 3 above)
Comment (abhisheksahani-nova) [reply src/pages/Notes/Notes.css:6]: Yaah sure , thanks for the review

**Pull request merged**